requirements/run.in

```diff
@@ -1,2 +1 @@
-git+https://github.com/amlight/scheduler@master#egg=amlight-scheduler
-git+https://github.com/amlight/flow_stats@master#egg=amlight-flow_stats
+requests
```

Comment:
@italovalcy thanks for the housekeeping in the requirements here, could you also pin the version `2.27.0` of `requests` like we have [on `coloring` on this PR that will land and also on `mef_eline`](https://github.com/amlight/coloring/pull/10/files) just so we have deterministic pip install resolution regardless of the order the NApps are installed?

```suggestion
requests==2.27.0
```

Reply:
Sounds great, @viniarck. I'll close this PR since it was already covered in PR #14.